src/tests/multihost/ad/conftest.py

```diff
@@ -7,14 +7,18 @@
 import pytest
 import os
 import posixpath
-from sssd.testlib.common.qe_class import session_multihost
```

Comment:
I already tried removing these lines in all the files and that made the tests fail. As far as I can see `session_multihost` is still used in `smbconfig()` and in several other places in this file. Did you try running any test that depends on this file to see if it's still working.

By the way, this comment applies to many of the changes from this patch set.

Reply:
The other option would be to move it to root conftest.py, which may break stuff depending on how the tests are called.

Reply:
Ok, I see now that it was moved. Thanks for the explanation.